boards/bcm2836/Packages/build.txt

```diff
@@ -0,0 +1,19 @@
+@echo off
+
+REM Run this command from the IoTCoreShell prompt only
```

Comment:
this is copied as build.cmd when we export the binaries ( binexport.cmd). required to build this wherever the binaries are exported.